### ISSUE TITLE
[🐞]: Network Protocol Error

### ISSUE THREAD

**Issue by chmonyasik** [MehVahdJukaar/Moonlight#279]
### BEFORE CONTINUING:

- [X] Always check you are using the latest version of these mods and its dependencies
- [X] Remove known problematic rendering mods such as Optifine, Sodium and others

### Version-Loader

1.21.1-NEOFORGE

### Moonlight Lib Version

moonlight-1.21-2.17.10-neoforge

### Describe the Issue

Unfortunately, I don't know exactly which mod this relates to. I tried to provide as many details as possible. This happens very strangely and not always. The position in the world and things from any nearby mod don't seem to affect it either. Randomly, one person might be able to log into the server, and after that, everyone else will be able to log in after

### Log Attachment

[disconnect-2024-12-04_22.05.28-client.txt](https://github.com/user-attachments/files/18013692/disconnect-2024-12-04_22.05.28-client.txt)

### OPTIONAL: To Produce

1) Server restart.
2) No one can log in due to this error.

### OPTIONAL: Which mods are affected?

Smarter Farmers (farmers replant)
Fast Paintings
Supplementaries
Every Compat (Wood Good)

**Comment by lockieluke** [comment]
getting this on Fabric too

**Comment by MehVahdJukaar** [comment]
does this stil happen?

**Comment by chmonyasik** [comment]
> does this stil happen?

It was a long time ago, so unfortunately, I don't know. However, I think if it hadn't been fixed or if it was really caused by some mod, someone else would have reported this issue by now

**Comment by MehVahdJukaar** [comment]
Yeah I thought the same, was probably fixed some time ago. I keep forgetting to close these
